### ISSUE TITLE
🐛 [Bug] SweetAlert - document is not defined

### ISSUE THREAD

**Issue by Sydney-o9** [Human-Connection/Human-Connection#1112]
Hi Team, running the webapp locally points to this error on master.

<img width="1645" alt="image" src="https://user-images.githubusercontent.com/1480617/61776034-7f179200-ae3d-11e9-9cc0-fb5eaaced1ac.png">

**Comment by mattwr18** [comment]
Thank you @Sydney-o9, we are looking into this, could you please let us know what steps you took to encounter the bug? 
### Steps to reproduce the behavior
1.
2.
3.
4. ...
5. Profit

**Comment by Sydney-o9** [comment]
Hi Team, 

I simply followed your doc here so there are no extra steps.

Meaning, I installed it without docker and use Node v10.15.0, NPM 6.4.1. 
Not necessarily important but I use Java v1.8.0_212 and neo4j 3.5.7.
1. Clone the repo
2. Copied and updated .env files in `neo4j` `backend` `webapp`
3. Run `npm install` and `npm run dev` in both `backend` and `webapp`

When commenting out `SweetalertIcon` logic in both `ConfirmModal.vue` and `ReportModal.vue`, it works well.

**Comment by roschaefer** [comment]
@Sydney-o9 you're using `npm` and there is no lockfile for npm because we use `yarn`. So npm installed a version of `vue-sweetalert-icon` for you which does not play nicely with server-side rendered applications.

FYI: I ran into the same issue when trying to update `vue-sweetalert-icon` and I created this:
https://github.com/JorgenVatle/vue-sweetalert-icons/issues/3

Seems that the maintainer already provided a solution - I'll give it a try.
